core/src/main/java/tech/tablesaw/api/DoubleColumn.java

```diff
@@ -841,4 +841,19 @@ private double subtract(double val1, double val2) {
         }
         return val1 - val2;
     }
+
+    /**
+    * Returns a new column with a cumulative sum calculated
```

Comment:
Can you format the javadocs throughout this PR with an extra space on the lines after the first like:
```
/**
 *
 */
```